dts/arm/st/f4/stm32f446.dtsi

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <zephyr/dt-bindings/clock/stm32f446_clock.h>
 #include <st/f4/stm32f401.dtsi>
 #include <zephyr/dt-bindings/clock/stm32f410_clock.h>
```

Comment:
I don't think including both bindings will work.

Reply:
Good catch. Removed the faulty commit